.gitignore

```diff
@@ -68,16 +69,7 @@ coverage.xml
 docs/_build/
 docs/_sources/
 docs/.doctrees
-docsource/modules50-60.rst
-docsource/modules60-61.rst
-docsource/modules61-70.rst
-docsource/modules70-80.rst
-docsource/modules80-90.rst
-docsource/modules90-100.rst
-docsource/modules100-110.rst
-docsource/modules110-120.rst
-docsource/modules120-130.rst
-docsource/modules130-140.rst
+docsource/modules*-*.rst
```

Comment:
This branch shouldn't contain any modules stuff. It should be catched from each branch, so I think you can remove it.